source/SpinalHDL/Data types/Floating.rst

```diff
@@ -23,54 +22,51 @@ This type can be used with the following import
 IEEE-754 floating format
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-The numbers are encoded into IEEE-754 `floating point format <https://en.wikipedia.org/wiki/IEEE_floating_point>`_.
+The numbers are encoded into IEEE-754 `floating-point format <https://en.wikipedia.org/wiki/IEEE_floating_point>`_.
 
 Recoded floating format
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Since IEEE-754 has some quirks about denormalized numbers and special values, Berkeley proposed another way of recoding floating
-point values.
+Since IEEE-754 has some quirks about denormalized numbers and special values, Berkeley proposed another way of recoding floating-point values.
 
 The mantissa is modified so that denormalized values can be treated the same as the normalized ones.
 
 The exponent field is one bit larger that one of the IEEE-754 number.
 
-The sign bit is kept unchanged between the two
-encodings.
+The sign bit is kept unchanged between the two encodings.
 
 Examples can be found `here <https://github.com/ucb-bar/berkeley-hardfloat/blob/master/README.md>`_
 
 Zero
 """"
 
-The zero is encoded with the three leading zeros of the exponent field being stuck to zero.
+The zero is encoded with the three leading zeros of the exponent field being set to zero.
 
 Denormalized values
 """""""""""""""""""
 
-Denormalized values are encoded in the same way as a normal floating point number. The mantissa is shifted so that the first one
-becomes implicit. The exponent is encoded as 107 (decimal) plus the index of the highest bit set to 1.
+Denormalized values are encoded in the same way as a normal floating-point number. The mantissa is shifted so that the first one becomes implicit.
+The exponent is encoded as 107 (decimal) plus the index of the highest bit set to 1.
 
 Normalized values
 """""""""""""""""
 
-The recoded mantissa for normalized values is exactly the same as the original IEEE-754 mantissa. The recoded exponent
-is encoded as 130 (decimal) plus the original exponent value.
+The recoded mantissa for normalized values is exactly the same as the original IEEE-754 mantissa. The recoded exponent is encoded as 130 (decimal) plus the original exponent value.
 
 Infinity
 """"""""
 
-The recoded mantissa value is treated as don't care. The recoded exponent three highest bits is 6 (decimal), the rest of the exponent can be treated as don't care.
+The recoded mantissa value is treated as don't care. The recoded exponent three highest bits is 6 (decimal), the rest of the exponent can be treated as don't cares.
```

Comment:
I think the don't care is correct. (based on this wiki page: https://en.wikipedia.org/wiki/Don%27t-care_term).
So maybe just use quotes? what do you think?
```suggestion
The recoded mantissa value is treated as don't care. The recoded exponent three highest bits is 6 (decimal), the rest of the exponent can be treated as "don't care".
```

Reply:
Either spelling works for me. I've heard folks around the lab where I work pluralize "don't care", so I carried that usage over to my edits out of habit.

I'll revert these to the original spelling, since I think the original is correct.